actions/approve_csr_handler_test.go

```diff
@@ -4,11 +4,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/google/uuid"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/google/uuid"
+
```

Comment:
We tried to group imports into 3 groups.
(
builtin

external

internal
)

Could you please join this with external imports?